docs/compose-server.md

```diff
@@ -80,6 +85,54 @@ Once we have that commit, let's export it:
 You can tell client systems to rebase to it by combining `ostree remote add`,
 and `rpm-ostree rebase` on the client side.
 
+## Granular tree compose with `install|postprocess|commit`
+
+In order to get even more control we split `rpm-ostree compose tree` into
+`rpm-ostree compose install`, followed by `rpm-ostree compose postprocess`
+and finally `rpm-ostree compose commit`.
+
+Similar to `rpm-ostree compose tree` we'll use a "treefile". We'll also specify a target directory
+serving as our work-in-progress rootfs:
+
+```
+# rpm-ostree compose install --unified-core --cachedir=cache --repo=/srv/repo /path/to/manifest.yaml /var/sysroot
+```
+
+This will download RPMs from the referenced repos and execute any specified post-process scripts.
+
+We now can alter anything found under `/var/sysroot/rootfs`.
+
+Next we can run more postprocessing:
+
+```
+# rpm-ostree compose postprocess postprocess /var/sysroot/rootfs /path/to/manifest.yaml
```

Comment:
```suggestion
# rpm-ostree compose postprocess /var/sysroot/rootfs /path/to/manifest.yaml
```

Reply:
Please note that this is stated ambiguously in the man pages:

```
rpm-ostree compose postprocess [OPTION…] postprocess ROOTFS [TREEFILE]
```

It is not clear what the second postprocess implies or what it stands for.

Reply:
Nice catch! Want to inline this? :)

```diff
diff --git a/src/app/rpmostree-compose-builtin-tree.cxx b/src/app/rpmostree-compose-builtin-tree.cxx
index 00da5c5e..5474f49e 100644
--- a/src/app/rpmostree-compose-builtin-tree.cxx
+++ b/src/app/rpmostree-compose-builtin-tree.cxx
@@ -1262,7 +1262,7 @@ rpmostree_compose_builtin_postprocess (int             argc,
                                        GCancellable   *cancellable,
                                        GError        **error)
 {
-  g_autoptr(GOptionContext) context = g_option_context_new ("postprocess ROOTFS [TREEFILE]");
+  g_autoptr(GOptionContext) context = g_option_context_new ("ROOTFS [TREEFILE]");
   if (!rpmostree_option_context_parse (context,
                                        postprocess_option_entries,
                                        &argc, &argv,
```